package.json

```diff
@@ -4,16 +4,18 @@
   "description": "UserALE.js is the UserALE client for DOM and JavaScript-based applications. It automatically attaches event handlers to log every user interaction on a web page, including rich JS single-page apps.",
   "main": "build/userale-2.4.0.js",
   "scripts": {
+    "format": "prettier --ignore-path .gitignore --write src/ test/",
     "lint": "eslint ./src --fix",
     "pretest": "npm run lint && npm run clean && npm run build",
-    "test": "mocha --require @babel/register && npm run journey:ci",
+    "test": "npm run pretest && mocha --require @babel/register && npm run journey:ci",
```

Comment:
Is there a reason you explicitly added pre-test here? It should automatically run whenever you call test. It's baked into the life cycle.